common/messages/messages_en.go

```diff
@@ -30,6 +30,20 @@ var messagesEn = map[string]util.Message{
 		RC:          400,
 		Action:      "Verify that you entered the correct IBM Cloud user name and password. If the error persists, the authentication service might be unavailable. Wait a few minutes and try again. ",
 	},
+	"EndpointNotReachable": {
+		Code:        "EndpointNotReachable",
+		Description: "Indicates IAM TOKEN exchange request failed.",
```

Comment:
For description can we say "IAM Token exchange request failed. Endpoint is unreachable"